MonacoEditorComponent/Monaco/Editor/IModelDecorationOverviewRulerOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace Monaco.Editor
+{
+    public sealed class IModelDecorationOverviewRulerOptions
+    {
+        /// <summary>
+        /// CSS color to render.
+        /// e.g.: rgba(100, 100, 100, 0.5) or a color from the color registry
+        /// </summary>
+        [JsonProperty("color", NullValueHandling = NullValueHandling.Ignore)]
+        public string Color { get; set; }
```

Comment:
Should we use a strongly typed C# Color class for this?

Reply:
Not sure if there are other cases like this to, just thought of it when I got to this class.

Reply:
Monaco introduced a type `ThemeColor` for all colors, here is the definition of `ThemeColor`:
```csharp
interface ThemeColor
{
    [JsonProperty("id", NullValueHandling = NullValueHandling.Ignore)]
    string Id { get; set; }
}
```

Reply:
I was thinking of just being able to use a C# Color class and do the translation during serialization, but I'll have to understand the new Monaco color system a bit more.  That's one of my next steps after getting this PR in, so I'll ramp up soon and investigate.